service/lib/dinstaller/storage/manager.rb

```diff
@@ -50,6 +51,7 @@ def initialize(config, logger)
       # Probes storage devices and performs an initial proposal
       def probe
         start_progress(4)
+        config.pick_product(software.selected_product)
```

Comment:
I don't like to have this here. Note that this class does not need the config at all. The config is used by `Storage::Proposal` and `Security`, so I would expect that classes retrieve the current config somehow. I think we need to re-think the config approach completely.

Reply:
https://github.com/yast/d-installer/issues/189

Reply:
Yes, I agree. I would keep the changes to the bare minimum, but it is clear that we need to evolve the configuration handling. I will add this information to #189.